lib/angular-evaporate.js

```diff
@@ -147,6 +157,11 @@
                       (100 - that.$percent) / that.$percent *
                       (now - that.$started) / 1000
                     );
+                    var progress = (arg * 100).toFixed(2);
+                    var delta = (new Date() - that.$started) / 1000;
+                    var loaded = (progress * that.sizeBytes) / 100;
+                    var totalSentBySecond = loaded / delta;
+                    that.$speed = readableFileSize(totalSentBySecond) + '/s';
```

Comment:
I think someone may need a custom format (e.g. for i18n purposes), so we'd better just calculate a speed here, say in bytes/second, and move that UI function from the internal scope to the default `ae.$config` so that it could be manually replaced. So, `that` will contain a pure `$speed` and a `$speedStr = that.speedStr(that.$speed)` or something like that.